QUEST_LV_0100.tsv

```diff
@@ -6403,101 +6403,101 @@ QUEST_LV_0100_20150317_006402	{memo X}Mr. Juan asked you to bring him Yukopus Le
-QUEST_LV_0100_20150323_006450	The teacher is poisoned and can't breathe. {nl}Quickly borrow antidote from others in Akmens Ridge. 
-QUEST_LV_0100_20150323_006451	I had no idea about Rukas Plateau. {nl}What are they trying to by releasing the seal? {nl}
+QUEST_LV_0100_20150323_006422	{memo X}You need to use the cable car to go from Srautas to Leaf Veil Plateau. {nl}I heard people used to hang baskets on ropes in the past. {nl}Well the cable cars are scary enough for me. 
+QUEST_LV_0100_20150323_006423	$I came to protect the divine land from the demons. {nl}I heard that the Paladin Master is also in Gele Plateau. It must surely be a big problem. {nl}
+QUEST_LV_0100_20150323_006424	$I'm worried for my brother in Nefritas Cliff. {nl}I hope nothing happens to him.{nl}
+QUEST_LV_0100_20150323_006425	$We'll have to manage with the remaining dolls left. {nl}Everyone's lives are at stake in Nefritas Cliff and we can't complain about this, right?
+QUEST_LV_0100_20150323_006426	$And if you're talking about Rukas Plateau, it's been a while since we pulled out of that place as it wasn't worthy of research anymore. 
+QUEST_LV_0100_20150323_006427	$Understanding the root of everything becomes the power of Elementalist. {nl}Wind, Water, Fire, Earth. We pull the strongest powers from the simplest forms. 
+QUEST_LV_0100_20150323_006428	$Bring the Unsealed Crystal Gem and find Disciple of Gustas in Ramstis Ridge. {nl}He will teach you about the seal. 
+QUEST_LV_0100_20150323_006429	$The Netherbovines are becoming violent again? {nl}As long as us Rodeleros are standing ground, no monsters shall pass.{nl}
+QUEST_LV_0100_20150323_006430	$I used to live in the woods but I came here to do what I had to. {nl}I will do whatever that can be of help.
+QUEST_LV_0100_20150323_006431	$Physical enhancing magic is a profound and mysterious field. {nl}If you make good use of it, you can easily identify friends and foes. 
+QUEST_LV_0100_20150323_006432	$If the seal is released, you must advance to the next seal in Ramstis Ridge.
+QUEST_LV_0100_20150323_006433	$Death may be the price that I pay for the sin I committed. {nl}Even if it is the curse of the demons, it is still for a sin I made.
+QUEST_LV_0100_20150323_006434	$I don't want to... die like this... {nl}Mother...
+QUEST_LV_0100_20150323_006435	$Wait. Aren't those Succubus Eyes? {nl}Wow. How did you get them... Can I take a look? 
+QUEST_LV_0100_20150323_006436	$I've heard about those eyes. {nl}Succubus craved the Cathedral's blessing that were hidden in a necklace, so she put it in her eyes. {nl}
+QUEST_LV_0100_20150323_006437	$Did you know I used to be an aspiring Alchemist? {nl}I can turn that back into a necklace. {nl}
+QUEST_LV_0100_20150323_006438	$Done. {nl}Do you know about the five Necklaces of Maven?{nl}
+QUEST_LV_0100_20150323_006439	$They were created to remind people of the 5 sins. {nl}Naktis caused them to look like curses on the Pilgrim Path.{nl}
```

Comment:
Previous line used "five" so
"remind people of the 5 sins." -> "remind people of the five sins."